.circleci/config.yml

```diff
@@ -175,51 +200,28 @@ commands:
           path: test-reports
           destination: test-reports
 
-  build-py-36-common:
-    steps:
-      - checkout
-
-      # Download and cache dependencies
-      # - restore_cache:
-      #     keys:
-      #       - v1-dependencies-{{ checksum "desktop/core/requirements.txt" }}
-      #       # fallback to using the latest cache if no exact match is found
-      #       - v1-dependencies-
-
-      - run:
-          name: compile
-          command: |
-            apt-get update && apt-get install -y python3.6-dev libsnappy-dev asciidoc # This should not be needed as some point
-
-            export PYTHON_VER=python3.6
-            make apps
-
-            make prod
-
-      - run:
-          name: run tests
-          command: |
-            PYTHONWARNINGS=always ./build/env/bin/hue test unit --with-xunit --with-cover
-
-      # - run:
-      #     name: install dependencies
-      #     command: |
-      #       python -m venv venv
-      #       make apps
-
-      # - save_cache:
-      #     paths:
-      #       - ./build/venv
-      #     key: v1-dependencies-{{ checksum "esktop/core/requirements.txt" }}
-
   build-py-38-common:
     steps:
       - checkout
 
       - run:
           name: compile
           command: |
-            apt-get install -y python3.8-dev python3.8-venv python3.8-distutils libsnappy-dev # This should not be needed as some point
+
+            if [ -d /usr/share/hue ]; then
+              # Running in gethue/hue Docker container
+              apt-get install -y python3.8-dev python3.8-venv python3.8-distutils libsnappy-dev # This should not be needed as some point
+              curl -sL https://bootstrap.pypa.io/get-pip.py | python3.8
+            else
+              export PATH=$HOME/.local/bin:$PATH
+              sudo ln -fs /usr/share/zoneinfo/UTC /etc/localtime
+              export DEBIAN_FRONTEND=noninteractive
```

Comment:
It is needed because otherwise the installation of `tzdata` package asks for interactive selection of a continent and country.
`tzdata` comes as a transitive dependency of one of the other packages.